components/button/button-split.js

```diff
@@ -0,0 +1,174 @@
+import './button.js';
+import '../colors/colors.js';
+import '../dropdown/dropdown.js';
+import '../dropdown/dropdown-menu.js';
+import '../icons/icon.js';
+import { css, html, LitElement } from 'lit';
+import { FocusMixin } from '../../mixins/focus/focus-mixin.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+/**
+ * A split button that provides a main action button and slot for a menu.
+ */
+class ButtonSplit extends FocusMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * @ignore
+			 */
+			// eslint-disable-next-line lit/no-native-attributes
+			autofocus: { type: Boolean, reflect: true },
```

Comment:
I don't know that autofocus ever worked...

Reply:
I can look into this more.